### DISCUSSION TITLE
Frequent occurrence X-Vercel-Error: ROUTER_EXTERNAL_TARGET_CONNECTION_ERROR

### DISCUSSION THREAD

**Discussion by xyfacai** [vercel/community#2840] [Bug Report]
### Page to Investigate

----

### Steps to Reproduce

I will deploy my website on Vercel for testing.
Packaging and deployment are normal, and the website appears to be running normally
But sometimes when fetching my API, it will make normal requests, and sometimes it will be abnormal, returning this error
X-Vercel-Error: ROUTER_EXTERNAL_TARGET_CONNECTION_ERROR

He has really troubled me for a long time, I don't understand why. I am an Hobby user.

### Steps to Reproduce

X-Vercel-Error: ROUTER_EXTERNAL_TARGET_CONNECTION_ERROR
![image](https://github.com/vercel/community/assets/52985669/241ef1b7-5555-448c-abc5-066176ceb2cb)
![image](https://github.com/vercel/community/assets/52985669/49900342-d382-4b89-a217-a761a20a234b)

It can be confirmed that this request did not request my backend service

This is my fetch code
```
  fetch() {
    if (fetchState > 0) return Promise.resolve();

    fetchState = 1;

    return fetch("backend/v1/user/account/info", {
      method: "get",
      body: null,
      headers: {
        ...getHeaders(),
      },
      credentials: "include",
    })
  },
```

This is my next.config.mjs
```
const nextConfig = {
  experimental: {
    appDir: true,
  },
  async rewrites() {
    const ret = [
      {
        source: "/google-fonts/:path*",
        destination: "https://fonts.googleapis.com/:path*",
      },
      {
        source: "/backend/:path*",
        destination: "http://123.456.789.789:80/backend/:path*",
      }
    ];
    
    return {
      beforeFiles: ret,
    };
  },
  webpack(config) {
    config.module.rules.push({
      test: /\.svg$/,
      use: ["@svgr/webpack"],
    });

    return config;
  },
  output: "standalone",
};

```

**Comment by smaeda-ks** [comment]
duplicate of https://github.com/orgs/vercel/discussions/2838
